### PR TITLE
[FSPE-5250] Improved chi.js global variable declaration

### PR DESCRIPTION
**Pull request by janibolkvadze**
@dani-cl-madrid @SergioQCL 

https://ctl.atlassian.net/browse/FSPE-5250

### CODE REVIEW

File: tasks/build-chi-scripts.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ const webpackConfig = {
   mode: WEBPACK_MODE,
   output: {
     library: 'chi',
-    filename: 'chi.js'
+    filename: 'chi.js',
+    libraryTarget: 'window'
     //libraryTarget: 'umd'
```
Comment (SergioQCL) [inline tasks/build-chi-scripts.js:23]: Is this comment still needed?

File: tasks/build-chi-scripts.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ const webpackConfig = {
   mode: WEBPACK_MODE,
   output: {
     library: 'chi',
-    filename: 'chi.js'
+    filename: 'chi.js',
```
Comment (dani-cl-madrid) [inline tasks/build-chi-scripts.js:21]: Will AMD way of instantiating Chi still work as it is in the documentation?
What is the improvement?

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ac8c80953209610b5871950fb4413029e235c068-chi-pr.enterprise.ctl.io" target="_blank"> https://ac8c80953209610b5871950fb4413029e235c068-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1071/ for more details.

File: tasks/build-chi-scripts.js (hunk 2 above)
Comment (janibolkvadze) [reply tasks/build-chi-scripts.js:21]: AMD is compiled by `build-chi-scripts-amd.js`. This change should not affect AMD.
The improvement is that currently chi global variable is declared like: `var chi = {}`. We encountered a compatibility issue with a Salesforce Lightning project which requires declarations like `window.chi = {}`.  If you look into `chi.js` compiled for this PR, you'll see the declaration change: https://ac8c80953209610b5871950fb4413029e235c068-chi-pr.enterprise.ctl.io/js/chi.js

File: tasks/build-chi-scripts.js (hunk 1 above)
Comment (janibolkvadze) [reply tasks/build-chi-scripts.js]: Removed, thanks ;)

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b2fb3d832c61e4741c0f357f43f533e8b21ba647-chi-pr.enterprise.ctl.io" target="_blank"> https://b2fb3d832c61e4741c0f357f43f533e8b21ba647-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1072/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-i4d5ho, https://ops.elasticbox.com/services/instances/i-hidcpv

**Pull request merged**